reason-react-native/src/components/DrawerLayoutAndroid.re

```diff
@@ -29,7 +29,7 @@ external make:
                        =?,
     ~onDrawerOpen: unit => unit=?,
     ~onDrawerSlide: drawerSlideEvent => unit=?,
-    ~onDrawerStateChanged: [ | `Idle | `Dragging | `Settling] => unit=?,
```

Comment:
bs.string can't be enough here?

Reply:
I couldn't get it to compile with the [@bs.string] annotation, but perhaps I'm missing how to properly annotate an inline type definition.

On the other hand, the callback function will be passed something like "Idle", etc. To act on that  we cannot make use of `[@bs.string]` as that's only good for `external` definitions and not `let` bindings.

I think this is exactly the same issue we've run into with other bindings and their callback functions.